src/commands/input.rs

```diff
@@ -0,0 +1,469 @@
+//! Implementation of the `input` command.
+
+use std::collections::HashSet;
+
+use anyhow::Result;
+use anyhow::bail;
+use clap::Parser;
+use serde_json::Map;
+use serde_json::Value;
+use wdl::analysis::types::CallKind;
+use wdl::ast::AstNode;
+use wdl::ast::AstToken;
+use wdl::ast::SyntaxKind;
+use wdl::ast::v1::Expr;
+use wdl::ast::v1::InputSection;
+use wdl::ast::v1::LiteralExpr;
+use wdl::ast::v1::Type;
+use wdl::cli::Analysis;
+use wdl::cli::analysis::AnalysisResults;
+use wdl::cli::analysis::Source;
+
+/// Arguments for the `input` subcommand.
+#[derive(Parser, Debug)]
+pub struct InputArgs {
+    /// The path to the WDL document for which to generate an input template.
+    #[arg(value_name = "PATH or URL")]
+    pub path: Source,
+
+    /// Task name for which to generate inputs.
+    #[arg(long, value_name = "TASK")]
+    pub task: Option<String>,
+
+    /// Show inputs with non-literal default values and non-required inputs.
+    #[arg(long)]
+    pub show_expressions: bool,
+
+    /// Hide inputs with default values.
+    #[arg(long)]
+    pub hide_defaults: bool,
+
+    /// Include task-level inputs.
```

Comment:
```suggestion
    /// Generate inputs for all tasks called in the workflow.
    ///
    /// When using this option may want to omit `--show-expressions` to avoid 
    /// cluttering the output with task-level inputs.
```

Then you can eliminate the long help I believe.